File: BonAppetitWeb/BonAppetitApp/Services/JsRuntimeServices/StripePaymentInvoke.cs
using Microsoft.JSInterop;

namespace Services.JsRuntimeServices;

public static class StripePaymentInvoke
{
    public static async ValueTask StripePayment(this IJSRuntime js, string sessionId)
    {
        await js.InvokeVoidAsync("stripeCheckout", sessionId);
    }
}

File: BonAppetitWeb/BonAppetitApp/BonAppetitWebApp/wwwroot/js/stripePaymentInvoke.js
stripeCheckout = function(sessionId) {
    var stripe = Stripe("pk_test_51KXsvmFJXN6PQCt7JfhEWI545YFipE5D5imBNjN3Qidd0Rc13nBSeonAEQB6ST4fucDwfMcEcfL0dwMilMguG3Uc00hLo2N173");

    stripe.redirectToCheckout({
         sessionId: sessionId
    });
};